brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaFacade.java

```diff
@@ -0,0 +1,41 @@
+package io.cloudsoft.tosca.a4c.brooklyn;
+
+import java.nio.file.Path;
+import java.util.Map;
+
+import org.apache.brooklyn.config.ConfigKey;
+
+import com.google.common.base.Optional;
+
+import alien4cloud.component.repository.exception.CSARVersionNotFoundException;
```

Comment:
You're right, I was aware, but couldn't decide what to do.  What do you think? A new Exception type or existing? Checked or unchecked?